noxfile.py

```diff
@@ -59,6 +59,31 @@ def unit(session):
         *session.posargs
     )
 
+@nox.session(python=UNIT_TEST_SYNC_PYTHON_VERSIONS)
```

Comment:
It seems unit() runs async + tests. and that this runs just tests. So I think maybe we have double runs?

Reply:
Changed this as we spoke about on our call.